### ISSUE TITLE
Permanent event text log

### ISSUE THREAD

**Issue by PJB3005** [space-wizards/space-station-14#190]
So while having everything in game with fancy hovering notifications is great, it's still nice to have a permanent log for a lot of things.

So ideally, there would be a permanent log of actions done by and surrounding you.

**Comment by PrPleGoo** [comment]
Make it so everything writes to a category message bus. That way people can define chat windows that include specific channels like "say" or "see".
>Aiden says: "Hello there sir"
>Aiden begins unscrewing the window pane from the floor.

**Comment by SaphireLattice** [comment]
Chat windows, or perhaps chat tabs or filters.

**Comment by tentekal** [comment]
PR #270 should address most of this, should we go further and set up an actual log file these events are written to?

**Comment by PaulRitter** [comment]
since this is kinda part of the logging issue/functionality, im gonna not it down as tech debt

**Comment by nok-ko** [comment]
#19364 closes this?

**Comment by AJCM-git** [comment]
> #19364 closes this?

kinda, the event log in ss13 is way more thorough with actions tho, for example  someone firing at you would have an event log entry. So popups should probably also have a "silent"/"event-log-only" param that doesn't display it on the screen but does log it for ones that would be annoying to display a lot of times like that one (btw, the firing one was just an example, pretty sure we don't actually want that logged here as it encourages unwanted behaviors) before this can be closed

**Comment by nok-ko** [comment]
Yeah, that or implement grouping for duplicate entries which is how some SS13 servers solved the problem in their chat systems.

**Comment by IProduceWidgets** [comment]
Given a PR that did what this says on its face was closed, and this feature request is so old its referenced by PRs in the three-digits, I'm electing to close it and direct you to make a design doc to further flesh out what this new feature should entail.
